src/SpecialPage/Special.php

```diff
@@ -282,16 +282,24 @@ private function generateCommentTable() : string {
 			$linkTitle = $sic->getPage();
 			if ( \ExtensionRegistry::getInstance()->isLoaded( 'DisplayTitle' )) {
 				$title = \Title::newFromText( $linkTitle );
-				if ( version_compare( MW_VERSION, '1.38', '>' ) ) {
-					$displaytitle = \MediaWiki\MediaWikiServices::getInstance()->getPageProps()->getProperties( $title, 'displaytitle' );
-				} else {
-					$displaytitle = \PageProps::getInstance()->getProperties( $title, 'displaytitle' );
+				$displaytitle = null;
+				if ( $title instanceof  \Title ) {
+					if ( version_compare( MW_VERSION, '1.38', '>' ) ) {
```

Comment:
@YvarRavy Deze version_compare mag nog weg, want we hebben in de composer gezegd dat het alles > MW1.39 is.